packages/studio-ui/src/web/views/FlowBuilder/nodeProps/ConditionModalForm.jsx

```diff
@@ -47,7 +47,7 @@ class ConditionModalForm extends Component {
 
     const { item } = this.props
     const condition = (item && item.condition) || ''
-    const conditionType = this.getConditionType(condition)
+    const conditionType = (item && item.conditionType) || this.getConditionType(condition)
```

Comment:
Optional chaining. item?.conditionType

Reply:
Since this is a JSX file, I guess we cannot use optional chaining!?

Reply:
If it doesn't, it would be nice to switch it to tsx :)